x/distribution/types/validator.go

```diff
@@ -50,3 +53,22 @@ func NewValidatorSlashEvent(validatorPeriod uint64, fraction sdk.Dec) ValidatorS
 		Fraction:        fraction,
 	}
 }
+
+func (vs ValidatorSlashEvent) String() string {
+	return fmt.Sprintf(`Period:   %d
+Fraction: %s`, vs.ValidatorPeriod, vs.Fraction)
+}
+
+// ValidatorSlashEvents is a collection of ValidatorSlashEvent
+type ValidatorSlashEvents []ValidatorSlashEvent
+
+func (vs ValidatorSlashEvents) String() string {
+	out := "Validator Slash Events:\n"
+	for i, sl := range vs {
```

Comment:
Should these have newlines appended?

Reply:
They are actually multiline strings and the newlines are added properly. Are you seeing issues when using that function?

Reply:
Im just thinking how will multiple slashes be separated?